src/state/opamSysInteract.ml

```diff
@@ -1093,14 +1105,51 @@ let install_packages_commands_t ?(env=OpamVariable.Map.empty) config sys_package
     [`AsUser (Commands.msys2 config),
      "-Su"::"--noconfirm"::packages], None
   | Netbsd -> [`AsAdmin "pkgin", yes ["-y"] ("install" :: packages)], None
+  | Nix ->
+    let open OpamFilename in
```

Comment:
I'd personally rather avoid opens. It makes it harder to understand the code
```suggestion
```

Reply:
I used them as I noticed their use in a fair bit of the rest of the codebase, but I'm happy to remove them if you prefer!